scripts/mod_loader/modapi/compat.lua

```diff
@@ -64,3 +64,18 @@ local statusMetatable = {
 	end
 }
 STATUS_TOOLTIPS = setmetatable(STATUS_TOOLTIPS, statusMetatable)
+
+-- ///////////////////////////////////////////////////////////////////////////////////
+-- compatibility code for KnightMiner's customPalettes
+local palettes = { VERSION = "1" }
+CUSTOM_PALETTES = palettes
```

Comment:
This is a little incomplete I think, as it won't prevent my library from calling it's finalize hook. I think there is just an internal boolean you need to set. I'll have to look at this more later to see if anything else is needed.